lib/copy_peste/command/init_bdd.rb

```diff
@@ -13,9 +13,9 @@ def run()
         config_path = File.join(Require::Path.root, './', 'copy_peste.yml')
         config_path = File.expand_path config_path
         config = YAML::load_file(config_path)
-        if File.directory? config['port_tree_path']
-          @graph_com.display(10, "Loading directory #{config['port_tree_path']}")
-          scan = ScanSystem.new(config['port_tree_path'])
+        if File.directory? config['ports_tree_path']
```

Comment:
I just discovered I had the changes locally but forgot to add them to the commit...